pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -89,6 +89,7 @@ public static void updateOrderStatus(BaseMercuryStatusMessage baseMercuryStatusM
                 int result = stmt.executeUpdate();
                 if (result != 0) {
                     log.info("Updated Mercury status for order id " + mercuryStatusMessage.getOrderID());
+                    baseMercuryStatusMessage.getStatus().setDetails(statusDetail);
```

Comment:
Would it make more sense to update this right after line 77? This feels like a non-sequitur.

Also, for the for the error cases below the error handling seems wrong. It looks like in one case we proceed with the second query after the first one fails, and what is the intention with the code just below? We log the error but proceed with the second query and the ES update? That case probably needs a comment if it is indeed the correct behavior.